lua/Lua.g4

```diff
@@ -71,7 +71,15 @@ stat
     | 'for' namelist 'in' explist 'do' block 'end'
     | 'function' funcname funcbody
     | 'local' 'function' NAME funcbody
-    | 'local' namelist ('=' explist)?
+    | 'local' attnamelist ('=' explist)?
+    ;
+
+attnamelist
+    : NAME attrib (',' NAME attrib)*
+    ;
+
+attrib
+    : ('<' NAME '>')?
```

Comment:
Maybe it's better to extract optional operator to uppermost rules?

Reply:
this is following the official manual.